data-processor/src/test/groovy/io/micronaut/data/processor/sql/ColumnTransformerSpec.groovy

```diff
@@ -152,6 +152,6 @@ class Project {
             def sql = builder.buildQuery(AnnotationMetadata.EMPTY_METADATA, QueryModel.from(entity).eq("xyz", new QueryParameter("xyz"))).query
 
         expect:
-            sql == 'SELECT transform_."project_id_department_id",transform_."project_id_project_id",UPPER(xyz@abc) AS xyz FROM "transform" transform_ WHERE (transform_."xyz" = LOWER(?))'
```

Comment:
One example where this looked wrong, should not have `project_id_` prefix for composite embedded id.